packages/venia-ui/lib/components/ProductOptions/swatch.js

```diff
@@ -40,9 +40,10 @@ const Swatch = props => {
 
     const { handleClick } = talonProps;
 
-    const icon = useMemo(() => {
-        return isSelected ? <Icon src={CheckIcon} /> : null;
-    }, [isSelected]);
+    const checkStyle = useMemo(
+        () => (isSelected ? defaultClasses.checked : defaultClasses.unchecked),
+        [isSelected]
+    );
```

Comment:
We want to reference `classes` rather than `defaultClasses` here. This means it should be moved after L48, and `classes` added to the deps array.

```
const classes = useStyle(defaultClasses, props.classes)

const checkStyle = useMemo(
    () => (isSelected ? classes.checked : classes.unchecked),
    [classes, isSelected]
)
```

Reply:
Ah, thanks for pointing that out. It's fixed now. Please review! :)